src/Dumplie/Application/Query/Customer/CartQuery.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Application\Query\Customer;
+
+use Dumplie\Application\Query\Customer\Result\Cart;
+
+interface CartQuery
+{
+    /**
+     * @param string $cartId
+     *
+     * @return Cart
+     */
+    public function getById(string $cartId) : Cart;
```

Comment:
I like to use gets that throws exception when resource does not exists, find sounds to me like "return object or null if not found" and I would like to avoid mixed results for queries